lib/backend/model/rule.go

```diff
@@ -61,6 +67,16 @@ type Rule struct {
 	NotDstNets     []*net.IPNet       `json:"!dst_nets,omitempty" validate:"omitempty"`
 	NotDstPorts    []numorstring.Port `json:"!dst_ports,omitempty" validate:"omitempty,dive"`
 
+	// These fields allow us to pass through the raw selectors from the V3 datamodel unmodified.
+	// The selectors above are formed in the update processor layer by combining the original
+	// selectors and namespace selectors into one.
```

Comment:
I think the Service Account selectors should be included in this block. Just like pod label selectors and namespace selectors, service account selectors are rolled into `SrcSelector` and `DstSelector`